pkgs/applications/science/math/mathematica/default.nix

```diff
@@ -35,9 +35,27 @@
 , xorg
 , zlib
 , lang ? "en"
+, cudaSupport ? false
+, nvidia_x11 ? null
+, cudatoolkit ? null
 }:
 
 let
+  cudaDeps = [
+    cudatoolkit
+    cudatoolkit.lib
+    (nvidia_x11.override { libsOnly = true; })
```

Comment:
```suggestion
```

Reply:
This breaks CUDALink.

Reply:
Not if you use `addOpenGLRunpath`. See the comments in this thread: https://github.com/NixOS/nixpkgs/pull/164333#discussion_r832495674.

Reply:
User-level derivations should not depend on kernel drivers like nvidia_x11.

Reply:
As I mentioned at the end of that thread, `addOpenGLRunpath` doesn't fix the problem.